partner_statement/tests/test_activity_statement.py

```diff
@@ -3,11 +3,14 @@
 
 from datetime import date
 
+from freezegun import freeze_time
+
 from odoo import fields
 from odoo.tests import new_test_user
 from odoo.tests.common import TransactionCase
 
 
+@freeze_time("2024-05-01", tick=True)
```

Comment:
Thanks, but I would put in `test_onchange_aging_type` below instead. The problem comes with that 30 days.

This  problem comes from
![Selection_3350](https://github.com/OCA/account-financial-reporting/assets/25005517/3e32662b-5348-4f29-b851-652ba3765c34)

Reply:
I don't see what you mean. Can you please propose the patch in other PR?

Reply:
Just freezing the specific test.

![Selection_3351](https://github.com/OCA/account-financial-reporting/assets/25005517/3ddd1c69-d796-4b93-99ad-a2ea0a5d40c1)